tests/test_directives.py

```diff
@@ -60,6 +75,14 @@ def test_default(self):
         assert MEMORY.default == 4
         assert PROCESS_FRACTION.default == 1.
 
+    def test_none_default_value(self, available_directives_list):
```

Comment:
I get that it is mostly miscommunication from me that you wrote this test, but instead of just testing None, testing a larger swath of invalid values should be done.